gumby/modules/tribler_module.py

```diff
@@ -273,7 +412,7 @@ def create_test_torrent(self, file_name, download_id, length):
         if not os.path.exists(file_name):
             self._logger.info("Creating torrent data file %s", file_name)
             with open(file_name, 'wb') as fp:
-                rand = Random()
+                rand = random.Random()
```

Comment:
**Suggestion:** It might be a good idea to seed the randomness in our experiments to make them (more) deterministic. I'll create a separate issue for this 👍

Reply:
Sure